tests/integration/aof.tcl

```diff
@@ -511,23 +511,15 @@ tags {"aof external:skip"} {
         }
     }
 
-    test {EVAL can't processes writes in AOF when doing "debug loadaof" in read-only slaves} {
-        start_server [list overrides [list dir $server_path appendonly no aof-use-rdb-preamble no]] {
-            r config set appendonly yes
-            waitForBgrewriteaof r
-            create_aof {
-                append_to_aof [formatCommand select 9]
-                append_to_aof [formatCommand eval {redis.call("set",KEYS[1],"100")} 1 foo]
-                append_to_aof [formatCommand eval {redis.call("incr",KEYS[1])} 1 foo]
-                append_to_aof [formatCommand eval {redis.call("incr",KEYS[1])} 1 foo]
-            }
-            r config set slave-read-only yes
-            r slaveof 127.0.0.1 0
-            r debug loadaof
-            after 100
-            r slaveof no one
+    test {EVAL can't process writes from AOF in read-only replicas} {
+        create_aof {
+            append_to_aof [formatCommand select 9]
+            append_to_aof [formatCommand eval {redis.call("set",KEYS[1],"100")} 1 foo]
+            append_to_aof [formatCommand eval {redis.call("incr",KEYS[1])} 1 foo]
+            append_to_aof [formatCommand eval {redis.call("incr",KEYS[1])} 1 foo]
+        }
+        start_server [list overrides [list dir $server_path appendonly yes aof-use-rdb-preamble no replica-read-only yes replicaof "1234 1234"]] {
             assert_equal [r get foo] ""
```

Comment:
I was setting it as "" instead of 102 to check if all tests run successfully to make sure that this is the only problem I'm going to deal with(and it fails with your suggestions, while passes with current commit). Currently I'm going to debug it to see what's going on on it.